setup.cfg

```diff
@@ -36,3 +37,10 @@ dev =
     %(test)s
     black
     flake8
+
+[flake8]
+max-line-length = 88
+# E203, W503 disabled because incompatible with black formatting
+# E262 (inline should begin by #) ignored because of numerous ## comments
+# E501 (line too long) ignored because of the end-of-line comments
+extend-ignore = E203, E262, E501, W503
```

Comment:
I added a sensible config for flake8. You can run flake8 to catch issues (such as undefined variable names or invalid formattion) using `flake8 [path_to_the_file]`.

With this config, `calibrationEDD.py` passes the `flake8` check. As I progress in the test writings, all files should pass the check.